pytest_tests/tests/object/test_object_api.py

```diff
@@ -52,6 +53,16 @@
 RANGE_MAX_LEN = 500
 # Used for static ranges found with issues
 STATIC_RANGES = {}
+OBJECT_NUMERIC_VALUES = [-1, 0, 1, 10]
```

Comment:
We need some `xyz` value as well to check that this object is never returned for numeric searches (attributes can have any values).

Some large (>2^64) values would also be nice to check.

Reply:
Do you mean this filter `attr GT xyz`? if we have letters instead of numbers we got `invalid search query` error, I have a test for that. 

For large value - added it.

Reply:
I mean searching for `attr GT 0`, but having an object with `attr=xyz`.

Reply:
added both scenarios